Ginger/GingerCore/ALM/OctaneCore.cs

```diff
@@ -226,5 +365,520 @@ public override bool IsServerConnected()
+        {
+            char[] lineBreaks = new char[] { '\n' };
+            QCTestCaseStepsColl stepsColl = new QCTestCaseStepsColl();
+            string steps = Task.Run(() => { return octaneRepository.GetTestCaseStep(GetLoginDTO(), testcase.Id); }).Result;
+            int i = 1;
+            if (!string.IsNullOrEmpty(steps))
+            {
+                foreach (string step in steps.Split(lineBreaks))
+                {
+                    stepsColl.Add(new QCTestCaseStep() { Id = Convert.ToString(i), TestId = testcase.Id, Description = step, Name = step, StepOrder = Convert.ToString(i++) });
+                }
+            }
+            return stepsColl;
+        }
+
+        private List<string> GetTCParameterList(string steps)
+        {
+            List<string> parameters = new List<string>();
+            string hasParams = @"(<[a-zA-Z0-9,+/-_()~!@#$%^&*=]+>)";
+            MatchCollection mc = Regex.Matches(steps, hasParams);
+            if (mc != null && mc.Count > 0)
+            {
+                foreach (Match m in mc)
+                {
+                    parameters.Add(m.Value.Substring(1, m.Value.Length - 2));
+                }
+            }
+            return parameters;
+        }
+
+        private void CheckForParameter(QC.QCTSTest newTSTest, string steps)
+        {
+            string hasParams = @"(<[a-zA-Z0-9,+/-_()~!@#$%^&*=]+>)";
+            MatchCollection mc = Regex.Matches(steps, hasParams);
+            if (mc != null && mc.Count > 0)
+            {
+                foreach (Match m in mc)
+                {
+                    QC.QCTSTestParameter newtsVar = new QC.QCTSTestParameter();
+                    newtsVar.Name = m.Value.Substring(1, m.Value.Length - 2);
+                    //if (paramValue != null) { newtsVar.Value = ; }                    
+                    newTSTest.Parameters.Add(newtsVar);
+                }
+            }
+        }
+
+        public QC.QCTSTest ImportTSTest(QCTestInstance testInstance)
+        {
+            QC.QCTSTest newTSTest = new QC.QCTSTest();
+            QCTestCase testCase = GetTestCases(testInstance.Id);
+            testCase.TestSetId = testInstance.CycleId;
+            if (testInstance != null)
+            {
+                //Regular TC
+                newTSTest.TestID = testInstance.Id;
+                newTSTest.TestName = testInstance.Name;
+            }
+
+            //Get the TC design steps
+            QCTestCaseStepsColl TSTestSteps = GetListTSTestSteps(testCase);
+            foreach (QCTestCaseStep testcaseStep in TSTestSteps)
+            {
+                QC.QCTSTestStep newtsStep = new QC.QCTSTestStep();
+                newtsStep.StepID = testcaseStep.Id.ToString();
+                newtsStep.StepName = testcaseStep.Name;
+                newtsStep.Description = testcaseStep.Description;
+                newtsStep.Expected = "";// testcaseStep.ElementsField["expected"].ToString();
```

Comment:
Codacy found an issue: [Remove this commented out code.](https://app.codacy.com/manual/Ginger-Automation/Ginger/pullRequest?prid=6317725)